src/loaders/common/loader_helper.py

```diff
@@ -46,6 +52,9 @@
 
 NONE_STR = ['N/A', 'NA', 'None', 'none', 'null', 'Null', 'NULL', '']
 
+HEATMAP_COL_PREFIX = "col"
+HEATMAP_COL_SEPARATOR = '_'
```

Comment:
Will these be needed by the service? If so they should go in common names

Reply:
I thought about this too. I intended to relocate it to common modules if it turns out we'll utilize it in the service. Otherwise, it will only be used for this module.

Reply:
:+1: